ui/Modal/Org/Create.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="typescript">
+  import type { Identity } from "ui/src/identity";
+
+  import { Copyable, Modal } from "ui/DesignSystem/Component";
+  import { Avatar, Button, Emoji } from "ui/DesignSystem/Primitive";
+
+  import * as error from "ui/src/error";
+  import * as org from "ui/src/org";
+  import { ellipsed } from "ui/src/style";
+  import * as modal from "ui/src/modal";
+
+  export let identity: Identity;
+  export let walletAddress: string;
+
+  const createOrg = async (owner: string): Promise<void> => {
+    modal.hide();
+    try {
+      await org.createOrg(owner);
+    } catch (err) {
+      error.show(
+        new error.Error({
+          code: error.Code.OrgCreatationFailed,
+          message: "Oops, something went wrong could not create the org",
```

Comment:
This sounds a bit odd. Maybe @brandonhaslegs could review the message?

Reply:
It's a catch-all in case _anything_ went wrong during the org creation phase. The user can click to copy the error and get in touch with us for debugging.

Reply:
If it's ambiguous and we do not know, try this: 

Something went wrong and the org wasn’t created. [Try again]

But would be cool to tell the user what went wrong.

Reply:
> But would be cool to tell the user what went wrong.

Agreed. We'll know more error cases once people run into them and report them to us via the "Copy error" button in the notification.

> Something went wrong and the org wasn’t created. [Try again]

I think we should inform the users to open an issue with the error text.
What would the `[Try again]` button do exactly?

Reply:
ca28e8375ae31393072a529b0cc5878669b08a52

I left it to `Something went wrong and the org wasn’t created`, with no period at the end to make it consistent with our other notifications.

Also didn't add the "try again" button, because it doesn't really make sense here.

Reply:
I ended up removing the outer error handling from the org Create screen and AnchorProject modal, because it would swallow more specific error messages like when the user rejects a transaction in their wallet.

b753848fdc37ed94b99a02c5f22154e6696f8d9f